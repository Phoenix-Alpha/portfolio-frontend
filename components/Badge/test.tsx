import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Badge } from '.'

describe('<Badge />', () => {
  it('should renders correct <Badge />', () => {
    const wrapper = shallow(<Badge title="Badge title" />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
