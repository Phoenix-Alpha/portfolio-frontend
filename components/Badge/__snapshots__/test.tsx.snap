// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> should renders correct <Badge /> 1`] = `
<span
  className="py-1 px-2 lg:px-3 m-1 text-sm font-normal text-white whitespace-nowrap bg-secondary-lighter hover:bg-[#6e6e6e] rounded-[3.125rem] transition duration-500"
>
  Badge title
</span>
`;
